roles/testconfig/tasks/main.yml

```diff
@@ -0,0 +1,143 @@
+---
+#- include_vars:
+#    file: images.yml
+#    name: images
+
+- name: source the overcloudrc file
+  shell: source /home/stack/overcloudrc
+
+- include: auth_old.yml
+- include: auth_demo.yml
```

Comment:
This include will not work for you in case auth_demo.yaml file still located at the root of the repo.

Reply:
It worked for me, where do you suggest to place it then, actually it failed in case that include isn't there

Reply:
When you define include, it will include a file when the file located in the same directory the main file located.
I didn't see you moved the auth* files into the role.

Am I missing something?